### DISCUSSION TITLE
Get badge

### DISCUSSION THREAD

**Discussion by learningssik** [Schweinepriester/github-profile-achievements#59] [Q&A] [answered]
can I get badge?

**Comment by lemonslapper** [comment, marked as answer]
Yes, first try something by selecting the next answer not this!
(test for https://github.com/Schweinepriester/github-profile-achievements/discussions/45#discussioncomment-3022366)

**Reply by nel0x** [reply]
#45

**Comment by nel0x** [comment]
This works fine:

**Comment by nel0x** [comment]
http://siteranker.com/

**Please select this answer as the solution @learningssik**

**Comment by nel0x** [comment]
Would you like to select one of my answers again as a solution @learningssik ?
Would be very beneficial otherwise what I wrote at https://github.com/Schweinepriester/github-profile-achievements/discussions/59 wouldn't make much sence anymore.

(You as the author of this post cannot get any badge out of it ...)

**Comment by Dentax** [comment]
test

**Comment by dexexpert** [comment]
Please mark this as answer!!!
Please 🙏

**Comment by abdelillahbel** [comment]
Hello, By accepting this answer you're legend! #45
